docs/advanced-ping/slots.md

```diff
@@ -0,0 +1,61 @@
+# Slots
+
+We launched a new way for you to interact with your embeds, they're called Slots!
+
+Slots are a way for you to have permanent Browser Sources from Ping, and be able to hot swap who goes where in your broadcast. This feature is great for those who leverage nested scenes, have different guests on your broadcast, or run a show where you need to move guests around mid-broadcast.

```

Comment:
> Slots are a way for you to have permanent Browser Sources from Ping

I don't love the phrasing "permanent browser sources", maybe something more like:

> Slots allow you to set up your browser sources in OBS once, and be able to how swap who is in them directly from Ping